### ISSUE TITLE
240p/Double Strike full screen support for Virtual Boy

### ISSUE THREAD

**Issue by niuus** [raz0red/wii-mednafen#78]
Is it possible?

**Comment by raz0red** [comment]
Yes, it should be possible, but the screen will not have the correct pixel aspect ratio.

The resolution is 384x224. So, if you double that it is 768x448. The best we could do would be 640x448 which may not look great. But, I can enable it for a build if you want to see how it looks.

**Comment by niuus** [comment]
Sure! I guess it would behave _sorta_ like the Virtual Tap, the video-out mod for the VB.  It outputs in “2x” 800×600 VGA mode (768×448) and RGB with a narrower aspect ratio, fitting the Virtual Boy’s 384×224 resolution into a 320×240 “240p” signal.
